extension/js/common/composer.ts

```diff
@@ -1286,6 +1283,11 @@ export class Composer {
     }
   }
 
+  private waitUntilRecipientsLoadedAndSetTestStateReady = async () => {
```

Comment:
I'll go ahead and rename this to `loadRecipientsThenSetTestStateReady`

Reply:
Ok, Thanks!